test/integration/targets/iam_user/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- name: Test that a user can be created, attached to a group, and then removed
+  block:
+    - name: ensure ansible user exists
+      iam_user:
+        name: AnsibleTestUser
+        state: present
+        <<: *aws_connection_info
+      register: iam_user
+
+    - name: assert that the user is newly created
+      assert:
+        that:
+          - iam_user is changed
+
+    - name: ensure group exists
+      iam_group:
+        name: ansible_test
+        users:
+          - AnsibleTestUser
+        state: present
+        <<: *aws_connection_info
+      register: iam_group
+
+    - assert:
+        that:
+          - iam_group.changed
+          - iam_group.iam_group.users
```

Comment:
I base this test on `iam_group` ones. This is different on https://github.com/ansible/ansible/blob/devel/test/integration/targets/iam_group/tasks/main.yml#L27 so I believe either my test is broken or `iam_group` module output is broken.

Documentation of `iam_group` goes in line with the `iam_group` test version, but not with the current module code..

Reply:
Unfortunately some of the tests **are** broken (including iam_group) see #61243